src/web/components/elements/ImageComponent.tsx

```diff
@@ -64,24 +69,65 @@ const getFallback: (imageSources: ImageSource[]) => string = imageSources => {
     return bestFor(300, inlineSrcSets).src;
 };
 
+const starsWrapper = css`
+    background-color: ${palette.brandYellow.main};
+
+    position: absolute;
+    ${until.tablet} {
+        bottom: 0;
+    }
+    ${from.tablet} {
+        top: 0;
+    }
+
+    /* Stars Padding from largest to smallest width */
+    ${from.leftCol} {
+        padding-left: 5px;
+    }
+
+    ${between.phablet.and.leftCol} {
+        padding-left: 0px;
+        margin-left: -0px;
+    }
+
+    ${between.mobileLandscape.and.phablet} {
+        padding-left: 10px;
+        margin-left: 0px;
+    }
+
+    ${until.mobileLandscape} {
```

Comment:
Could this be merged with the above to be `until.phablet`?

Reply:
I dnt see how, everything under and including `mobileLandscape` should have this style, the rest is about everything in-between, am I missing something? could you reference the line in question?

Reply:
I think this:

```
// including mobileLandscape up to and not including phablet
${between.mobileLandscape.and.phablet} {
	padding-left: 10px;
	margin-left: 0px;
}

// up to and not including mobileLandscape
${until.mobileLandscape} {
	padding-left: 10px;
	margin-left: 0px;
}
```

is the same as this:

(up to and not including phablet)

```
${until.phablet} {
	padding-left: 10px;
	margin-left: 0px;
}
```

?

I think `until` does not include that breakpoint https://github.com/guardian/source/blob/99428b18e2a2418425c843f0c2d1e4ff53d1ba60/src/core/foundations/src/mq/index.ts#L22 and `between.` does include the breakpoint `until`

Does that make sense?